pikaraoke/lib/get_platform.py

```diff
@@ -24,15 +24,25 @@ def get_ffmpeg_version():
 def is_raspberry_pi():
     try:
         return (
-            os.uname()[4][:3] == "arm" or os.uname()[4] == "aarch64"
-        ) and sys.platform != "darwin"
+            (os.uname()[4][:3] == "arm" or os.uname()[4] == "aarch64")
+            and sys.platform != "darwin"
+            and not (os.path.exists("/system/app/") and os.path.exists("/system/priv-app"))
+        )
     except AttributeError:
         return False
 
```

Comment:
How about wrapping this in a function so we don't duplicate this code in the file:

```
def is_android():
  return os.path.exists("/system/app/") and os.path.exists("/system/priv-app")
```